pkg/controller/oneagent/util.go

```diff
@@ -153,7 +153,7 @@ func getPodsToRestart(pods []corev1.Pod, dtc dtclient.Client, instance *dynatrac
 		item := dynatracev1alpha1.OneAgentInstance{
 			PodName: pod.Name,
 		}
-		ver, err := dtc.GetVersionForIp(pod.Status.HostIP)
+		ver, err := dtc.GetAgentVersionForIP(pod.Status.HostIP)
```

Comment:
GetAgentVersionForIP is missing logs in implementation as well as function call. 
Must revisit `getPodsToRestart` for logging (pls ignore if already done in base branch).